mtt/config/Run2/config_2017.py

```diff
@@ -373,8 +376,140 @@
         dataset.add_tag({"is_e_data", "is_egamma_data"})
     if dataset.name.startswith("data_pho"):
         dataset.add_tag({"is_pho_data", "is_egamma_data"})
-    if dataset.name.startswith("data_pho"):
-        dataset.add_tag({"is_pho_data", "is_egamma_data"})
+
+#
+# tagger configuration
+# (b/top taggers)
+#
+
+# 2017 b-tagging working points
+# https://twiki.cern.ch/twiki/bin/view/CMS/BtagRecommendation106XUL17?rev=15
+config_2017.x.btag_wp = {
+    "deepjet": {
+        "loose": 0.0532,
+        "medium": 0.3040,
+        "tight": 0.7476,
+    },
+    "deepcsv": {
+        "loose": 0.1355,
+        "medium": 0.4506,
+        "tight": 0.7738,
+    },
+}
+
+# 2017 top tagging working points (DeepAK8, 1% mistagging rate, )
+# https://twiki.cern.ch/twiki/bin/viewauth/CMS/DeepAK8Tagging2018WPsSFs?rev=4
+config_2017.x.toptag_wp = {
+    "deepak8": {
+        # regular tagger
+        "top": 0.344,
+        "w": 0.739,
+        # mass-decorrelated tagger
+        "top_md": 0.725,
+        "w_md": 0.925,
+    },
+}
+
+#
+# selector configuration
+#
+
+# lepton selection parameters
+config_2017.x.lepton_selection = DotDict.wrap({
+    "mu": {
+        "column": "Muon",
+        "min_pt": {
+            "low_pt": 30,
+            "high_pt": 55,
+        },
+        "max_abseta": 2.4,
+        "iso": {
+            "column": "pfIsoId",
+            "value": 4,
```

Comment:
Since the `pfIsoId` only contains a single value per muon, corresponding to the tightest working point passed, one should adapt the selection code to also select muons that have `pfIsoId` value greater than or equal to the one on the config (i.e. pass a tighter selection than the value given here). For clarity, it would be good to reflect this in the config key name, so I would suggest to change this from `value` to `min_value`:
```suggestion
            "min_value": 4,
```